core/health/api/src/main/java/org/opennms/core/health/api/HealthCheck.java

```diff
@@ -44,6 +44,14 @@
      */
     String getDescription();
 
+    /**
+     *
+     * A tag to indicate the category of the health-check.
+     *
+     * @return The HealthTag type, describing category of the health-check.
+     */
+    HealthTag getTag();
```

Comment:
As we discussed before, change this to   `List<String> getTags()`.  That would give more flexibility for implementations to choose tags.

Reply:
How about use List<HealthTag> ?

Reply:
we can stick to strings to expose those as constants here in this Interface.

Reply:
Sure. Updated. If you don't mind. I use a HealthConstant class for the constants instead of using Constant Interface.

Reply:
> Sure. Updated. If you don't mind. I use a HealthConstant class for the constants instead of using Constant Interface.

Not sure if there is any benefit of using Class. HealthCheckConstants would be better name for it.